python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -332,8 +335,8 @@ def _set_run_tags(self, run_id: str, cluster: str, task_arn: str):
         }
         self._instance.add_run_tags(run_id, tags)
 
-    def build_ecs_tags_for_run_task(self, run):
-        return [{"key": "dagster/run_id", "value": run.run_id}]
+    def build_ecs_tags_for_run_task(self, run, container_context: EcsContainerContext):
+        return [{"key": "dagster/run_id", "value": run.run_id}, *container_context.run_ecs_tags]
```

Comment:
this is the part where we actually read the value off of the container context object and use it to do things to the task